mod/metrics/set/type_plus_right/metric/formula.rb

```diff
@@ -78,11 +78,11 @@ def add_metric_button
   view :core do
     return _render_rating_core if card.wiki_rating?
     return _render_categorical_core if card.categorical?
-    "= #{super()}"
+    "<span>=</span><span>#{super()}"
```

Comment:
fixed it and it will be available in my next push (along with source preview page related updated )